packages/core/src/payment/strategies/paypal-commerce/paypal-commerce-payment-strategy.ts

```diff
@@ -46,6 +47,7 @@ export default class PaypalCommercePaymentStrategy implements PaymentStrategy {
         const { orderId, buttonStyle, shouldRenderFields } = initializationData ?? {};
         this._isAPM = gatewayId === PaymentStrategyType.PAYPAL_COMMERCE_ALTERNATIVE_METHODS;
         this._isPaylater = methodId === PaymentStrategyType.PAYPAL_COMMERCE_CREDIT;
+        this._isVenmo = methodId === PaymentStrategyType.PAYPAL_COMMERCE_VENMO && initializationData.isVenmoEnabled;
```

Comment:
It might be better to make destructurization of initializationData and ge isVenmoEnabled value like this.

```suggestion
        const { orderId, buttonStyle, isVenmoEnabled, shouldRenderFields } = initializationData ?? {};
        this._isAPM = gatewayId === PaymentStrategyType.PAYPAL_COMMERCE_ALTERNATIVE_METHODS;
        this._isPaylater = methodId === PaymentStrategyType.PAYPAL_COMMERCE_CREDIT;
        this._isVenmo = methodId === PaymentStrategyType.PAYPAL_COMMERCE_VENMO;
```